miso-web/src/main/webapp/scripts/hot_sample.js

```diff
@@ -293,9 +297,35 @@ HotTarget.sample = (function() {
               flat.externalName = Utils.valOrNull(sam.externalName);
             },
             pack: function(sam, flat, errorHandler) {
+              sam.externalNames = flat.externalName;
               if (!getSelectedIdentity(flat)) {
                 sam.externalName = flat.externalName;
               } // else externalName will come from an existing Identity via the Identity Alias column
+            },
+            depends: 'identityAlias',
+            update:function(sam, flat, flatProperty, value, setReadOnly, setOptions, setData) {
+              if(!value || value == "Delete external name, select a project, then re-enter external name." || value == '(...searching...)'
+                || /First Receipt \(.*\)/.test(value)){
+                setOptions({
+                  'renderer': Handsontable.renderers.TextRenderer
+                });
+                return;
+              };
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this empty statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS1116)